plot_goes_x.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
Include license and then a description of the program as a docstring.

Reply:
Docstring or comment?  I assume we should just copy the standard LICENSE.rst into the project and comment all the files or some such?